backend/supabase/groups/indexes.sql

```diff
@@ -0,0 +1,5 @@
+
+create index if not exists group_slug on public.groups (slug);
+create index if not exists group_name on public.groups (name);
+create index if not exists group_creator_id on public.groups (creator_id);
+create index if not exists total_members on public.groups (total_members desc);
```

Comment:
It looks like you're searching `name_fts` in non-fuzzy search, here's the query plan using the dev db:
<img width="921" alt="Screenshot 2023-05-16 at 9 18 19 AM" src="https://github.com/manifoldmarkets/manifold/assets/23196210/d0b378ea-0782-4ee1-85e0-6934f19da35d">